clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/DeleteExecutor.scala

```diff
@@ -4,12 +4,12 @@ import java.net.URI
 
 import akka.NotUsed
 import akka.stream.scaladsl.Source
+import cats.syntax.show._
```

Comment:
Is the wildcard import needed if we're only calling `show`?

Reply:
There's probably a specific implicit hook we could pull by name here, but this is the recommended style when pulling in implicit syntax.